### PR TITLE
feat(FN-2353): refactor usages of report statuses to be from the common library

### PR DESCRIPTION
**Pull request by RLavender98**
## Introduction :pencil2:
Remove duplicate definitions of utilisation report status related types defined in the common lib.

## Resolution :heavy_check_mark:
- Ensure all imports of `UTILISATION_REPORT_RECONCILIATION_STATUS` come from `libs/common`
- Ensure all imports of `UtilisationReportReconciliationStatus` come from `libs/common`
- Ensure all imports of `ReportWithStatus` come from `libs/common`

## Miscellaneous :heavy_plus_sign:
- Some of the old `ReportsWithStatus` where using string not number for the report id so have cast this to a string in tfm ui at the earliest convenience and then used number for the report id everywhere for consistency.

### CODE REVIEW

File: trade-finance-manager-ui/server/controllers/utilisation-reports/update-utilisation-report-status/index.ts [hunk 1]
```diff
@@ -1,8 +1,7 @@
 import { Response } from 'express';
-import { UtilisationReportReconciliationStatus, UTILISATION_REPORT_RECONCILIATION_STATUS } from '@ukef/dtfs2-common';
+import { UtilisationReportReconciliationStatus, UTILISATION_REPORT_RECONCILIATION_STATUS, ReportWithStatus } from '@ukef/dtfs2-common';
```
Comment (francescastocco) [inline trade-finance-manager-ui/server/controllers/utilisation-reports/update-utilisation-report-status/index.ts:2]: Same as above please

**Review by francescastocco — COMMENTED** [review]
Looks good, just some really minor import formatting

File: portal-api/src/cron-scheduler-jobs/helpers/utilisation-report-helpers.test.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { BankReportPeriodSchedule, UtilisationReportReconciliationStatus } from '@ukef/dtfs2-common';
+import { BankReportPeriodSchedule, UtilisationReportReconciliationStatus, UTILISATION_REPORT_RECONCILIATION_STATUS } from '@ukef/dtfs2-common';
```
[the remark this answers is not in the document]
Comment (RLavender98) [reply portal-api/src/cron-scheduler-jobs/helpers/utilisation-report-helpers.test.ts:1]: If I manually make this change then run prettier it forces it back onto one line, so I think we are stuck with it 😞

**Review by francescastocco — APPROVED** [review]
LGTM!

**Pull request merged**